lib/src/collection/treeset.dart

```diff
@@ -71,12 +71,15 @@ abstract class TreeSet<V> extends IterableBase<V> implements Set<V> {
 /// Controls the results for [TreeSet.searchNearest]()
 enum TreeSearch {
   /// If result not found, always chose the smaller element
+  // ignore: constant_identifier_names
```

Comment:
Do we have generalized enums yet? We could introduce new names and redirect the old ones (as deprecated) using static variables. One caveat is it would change the result of `toString`.

Reply:
We could!

Reply:
Maybe in a follow-up?

Reply:
SGTM